reference/safe-core-sdk/onramp-kit/monerium.md

```diff
@@ -0,0 +1,151 @@
+## Monerium
+
+The Monerium pack enables you to take advantage of Safe with Monerium services. [Monerium](https://monerium.com) is a regulated platform that facilitates the use of e-money tokens on the blockchain.
+
+### Install
+
+If you choose to use the `MoneriumPack`, you must install additional dependencies in addition to the `@safe-global/onramp-kit` package.
+
+Install the `@monerium/sdk` package:
```

Comment:
I don't think we need to explain this part, it's self-explanatory given the title and commands that follow.

```suggestion
```